site/en/gemini-api/docs/prompting_with_media.ipynb

```diff
@@ -65,10 +65,9 @@
       },
```

Comment:
<p>Can we make this wording sound less strict?</p><blockquote>they are subject to the following limitations and requirements</blockquote><p> </p><p>It's not a strict requirement that plain text inputs adhere to one of these formats, since they don't map to an encoding like other media.</p><p> </p><p>We should also recommend that users specify the MIME type manually if it isn't in this list. In <a href="https://github.com/google-gemini/cookbook/pull/104" target="_blank">this cookbook recipe</a> I gave an example with a C++ file because that's not detected as plain-text, but it really is, so users can specify it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>